src/sql/workbench/services/dialog/browser/wizardModal.ts

```diff
@@ -268,9 +268,11 @@ export class WizardModal extends Modal {
 	}
 
 	public cancel(): void {
+		//currentPage title will be used for Telemetry
```

Comment:
Just remove this comment - it's clear enough from the code what this will be used for